### PR TITLE
chore: upgrade error-stack version and remove deprecated dependencies

### PR DESCRIPTION
**Pull request by cgorenflo**

### CODE REVIEW

File: ampd/src/broadcaster/accounts.rs [hunk 1]
```diff
@@ -30,13 +30,14 @@ where
...
         .account
-        .ok_or_else(|| Error::AccountNotFound {
-            address: address.clone(),
+        .ok_or_else(|| {
+            Error::AccountNotFound {
+                address: address.clone(),
+            }
+            .into()
         })
-        .into_report()
```
Comment (sammy1991106) [inline ampd/src/broadcaster/accounts.rs:36]: One of the other reasons I didn't upgrade was also that I found `.into_report` quite nice haha. Anyway.

File: packages/report/src/result_ext.rs [hunk 2]
```diff
@@ -0,0 +1,22 @@
+use error_stack::{Context, IntoReportCompat, Report};
+use eyre::Report as EyreReport;
+
+pub trait ResultCompatExt {
```
Comment (sammy1991106) [inline packages/report/src/result_ext.rs:4]: nice!

File: packages/axelar-wasm-std/src/lib.rs [hunk 3]
```diff
@@ -3,6 +3,7 @@ pub mod flagset;
 mod fn_ext;
 pub mod nonempty;
 pub mod operators;
+
```
Comment (sammy1991106) [inline packages/axelar-wasm-std/src/lib.rs:6]: Any reason for adding this empty line? Just curious

**Review by sammy1991106 — APPROVED** [review]

File: packages/axelar-wasm-std/src/lib.rs (hunk 3 above)
Comment (cgorenflo) [reply packages/axelar-wasm-std/src/lib.rs:6]: might be that the error-stack import was on that line and I only deleted the text, not the whole line

**Review by haiyizxx — APPROVED** [review]

**Pull request merged**